plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/ForteNgExportTemplate.xtend

```diff
@@ -37,11 +37,11 @@ abstract class ForteNgExportTemplate extends ExportTemplate {
 		«FOR include : types.map[generateDefiningInclude].toSet.sort»
 			«include.generateDependencyInclude»
```

Comment:
@mx990 @MartinMelikMerkumians  do you have a tip for me how I can adjust this loop here to also give standard data type includes the `core/datatypes/` prefix.

Reply:
I debugged a bit and found a place where it should not have to much impact on other parts. I hope this is correct

Reply:
Today in the morning I got curios and wanted to better understand our code generator. Happy to hear that I found the right spot.